bats/tests/registry/creds.bats

```diff
@@ -161,6 +162,26 @@ verify_default_credStore() {
     rdsudo update-ca-certificates
 }
 
+restart_container_engine() {
+    # BUG BUG BUG
+    # When using containerd, sometimes the container would get wedged on a
+    # restart; however, restarting containerd again seems to fix this.
+    # So we need to keep trying until the registry container is not `created`.
+    # BUG BUG BUG
```

Comment:
The comment would be better on top of the `if using_containerd` block, but good enough as is.